spec/models/manageiq/providers/ibm_power_hmc/infra_manager/host_spec.rb

```diff
@@ -0,0 +1,41 @@
+describe ManageIQ::Providers::IbmPowerHmc::InfraManager::Host do
+  let(:ems) do
+    username = Rails.application.secrets.ibm_power_hmc[:username]
+    password = Rails.application.secrets.ibm_power_hmc[:password]
+    hostname = Rails.application.secrets.ibm_power_hmc[:hostname]
+
+    FactoryBot.create(:ems_ibm_power_hmc_infra, :endpoints => [FactoryBot.create(:endpoint, :role => "default", :hostname => hostname, :port => 12_443)]).tap do |ems|
```

Comment:
If you're going to use this more than once, you might consider creating an `ems_ibm_power_hmc_infra_with_vcr_authentication` factory where you set this all up for you (e.g. https://github.com/ManageIQ/manageiq-providers-azure/blob/master/spec/factories/ext_management_system.rb#L2)

Reply:
Looks like a default endpoint object is created with random values (e.g. ems-000001) before we have a chance to add our own. Trying:
```
    endpoints do
      [FactoryBot.create(:endpoint, :role => "default", :hostname => hostname, :port => 12_443)]
    end
    
    after(:create) do |ems|
      ems.authentications << FactoryBot.create(:authentication, :userid => username, :password => password)
    end
```